### DISCUSSION TITLE
`No Space Left On Device`/`Disk Space Issue`

### DISCUSSION THREAD

**Discussion by AlexanderMann** [renovatebot/renovate#14503] [Request Help]
# Motivation

Working in a private repo for the MDG Private Github Org. The most recent build number exhibiting this issue is 592615050

Saw that we should reach out to you all from here: https://github.com/renovatebot/app-support/issues/116

Just to take the moment: we really enjoy this product, and are constantly impressed by everything you all are doing. Thanks!

**Comment by viceice** [comment]
Had the same issue recently. Currently your only option is to clear the huge renovate file cache from time to time or use a Redis.

**Reply by AlexanderMann** [reply]
Ah, we're still using the public version, not a self hosted instance. I'm assuming clearing a cache/using redis is only for self hosted yeah?

**Reply by rarkins** [reply]
I've configured this repo to use the bigger/slower EBS instead of SSD

**Reply by AlexanderMann** [reply]
Looks to have fixed things! Thanks @rarkins !

**Comment by mhagnumdw** [comment]
Is there currently an automatic way for Renovate Bot self-hosted to clear the cache? I have 100 GB reserved for cache in the file system, and it fills up from time to time.

```
--- /pvc/cache/renovate/renovate-cache-v1 ----
                           /..
   88.1 GiB [############] /content-v2
    1.3 GiB [            ] /index-v5
e   4.0 KiB [            ] /tmp
```

**Reply by viceice** [reply]
use sqlite or redis cache

**Reply by jamietanna** [reply]
I think generally we don't have anything inbuilt to clear this out on a schedule?

Does Renovate periodically delete anything that's expired in the on-disk cache, if it notices that it's stale?

**Reply by viceice** [reply]
the package cache should be cleaned on each run when elements expire. redis and sqlite caches are a lot smaller thant the old cacache

**Reply by jamietanna** [reply]
Closing and locking as this is an old thread we shouldn't be bumping - if you're still looking for support, mind raising a fresh Discussion? Thanks!
